README.md

```diff
@@ -573,3 +574,157 @@ export class MyAwesomeComponent extends React.PureComponent {
     }
 }
 ```
+## External libraries
+
+### Prefer native js vs lodash
+
+> Reason: lodash is a great library which contains a lot of useful functions but the modern js has the short identical part of this functionality so some lodash functions are excess alternative.
+
+```javascript
+// BAD
+_.each([1, 2, 3], callback); 
+// GOOD
+[1, 2, 3].forEach(callback); 
+```
+```javascript
+// BAD
+_.filter([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].filter(callback);
+```
+```javascript
+// BAD
+_.compact([1, 2, 3, 0, false, '']);
+// GOOD
+[1, 2, 3, 0, false, ''].filter(Boolean)
+```
+```javascript
+// BAD
+_.map([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].map(callback);
+```
+```javascript
+// BAD
+_.every([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].every(callback);
+```
+```javascript
+// BAD
+_.some([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].some(callback);
+```
+```javascript
+// BAD
+_.reduce([1, 2, 3], callback);
+// GOOD
+[1, 2, 3].reduce(callback);
+```
+```javascript
+// BAD
+_.concat([1, 2, 3], [4, 5, 6]);
+// GOOD
+[1, 2, 3].concat([4, 5, 6]);
+```
+```javascript
+// BAD
+_.indexOf([1, 2, 3], 1);
+// GOOD
+[1, 2, 3].indexOf(1);
+```
+```javascript
+// BAD
+_.includes([1, 2, 3], 1);
+// GOOD
+[1, 2, 3].includes(1);
+```
+```javascript
+// BAD
+const first = _.first([1, 2, 3]);
+// GOOD
+const [first] = [1, 2, 3];
+```
+```javascript
+// BAD
+_.join([1, 2, 3], '-');
+// GOOD
+[1, 2, 3].join('-');
+```
+```javascript
+// BAD
+_.reverse([1, 2, 3]);
+// GOOD
+[1, 2, 3].reverse();
+```
+```javascript
+// BAD
+_.isArray([1, 2, 3]);
+// GOOD
+Array.isArray([1, 2, 3]);
+```
+```javascript
+// BAD
+_.size([1, 2, 3]);
+// GOOD
+[1, 2, 3].length;
+```
+```javascript
+// BAD
+_.bind(function, this);
+// GOOD
+function.bind(this);
+```
+```javascript
+// BAD
+_.isNaN(value);
+// GOOD
+Number.isNaN(value);
+```
+```javascript
+// BAD
+_.isUndefined(value);
+// GOOD
+value === undefined;
```

Comment:
please add example regarding to `_.isFunction` - it's still preferred way

Reply:
Maybe we should avoid `isFunction` too? It's reliable to use `typeof === 'function'`; We have the rule https://eslint.org/docs/rules/valid-typeof to avoid typos.